.github/workflows/reusable_deploy_v2_sar.yml

```diff
@@ -67,13 +78,28 @@ jobs:
       matrix:
         architecture: ["x86_64", "arm64"]
     steps:
-      - name: Checkout
+      - name: checkout
```

Comment:
Was this an intentional change?

Reply:
ah copy/paste from other workflows to use the release commit, I'd leave it as-is